kolibri/plugins/device_management/assets/src/views/ManageContentPage/ChannelsGrid.vue

```diff
@@ -58,11 +57,12 @@
       ChannelListItem,
       KLinearLoader,
       DeleteChannelModal,
-      KButton,
     },
-    data: () => ({
-      selectedChannelId: null,
-    }),
+    data() {
+      return {
+        selectedChannelId: null,
+      };
+    },
```

Comment:
is this style enforced now? I think we've used both styles in the code base, but I do prefer this one

Reply:
I just changed them whenever I saw them. I prefer them too just in case we need to access `this`